internal/daemonset/daemonset.go

```diff
@@ -214,13 +213,12 @@ func (dc *daemonSetGenerator) SetDriverContainerAsDesired(
 				Annotations: modulesOrderAnnotations,
 			},
 			Spec: v1.PodSpec{
-				ShareProcessNamespace: swag.Bool(true),
-				Containers:            []v1.Container{container},
-				ImagePullSecrets:      GetPodPullSecrets(mld.ImageRepoSecret),
-				NodeSelector:          nodeSelector,
-				PriorityClassName:     "system-node-critical",
-				ServiceAccountName:    mld.ServiceAccountName,
-				Volumes:               volumes,
+				Containers:         []v1.Container{container},
+				ImagePullSecrets:   GetPodPullSecrets(mld.ImageRepoSecret),
+				NodeSelector:       nodeSelector,
+				PriorityClassName:  "system-node-critical",
+				ServiceAccountName: mld.ServiceAccountName,
+				Volumes:            volumes,
```

Comment:
yes. As mentioned in the commit/PR message, ShareProcessNamespace was introduced in order to solve 30 second "Terminating state" for Module Loader pods. But it means that the update controller(v1.1) will change the existing ModuleLoader's Container spec, which will cause the removal and re-creation of a pod on a node. So, i prefer to skip it, since the 30 seconds issue is not a critical one

Reply:
Sorry for missing that in the description.